modules/apps/portal-vulcan/portal-vulcan-impl/src/main/java/com/liferay/portal/vulcan/internal/jaxrs/context/provider/ContextProviderUtil.java

```diff
@@ -107,6 +119,30 @@ public static MultivaluedHashMap<String, String> getMultivaluedHashMap(
 		};
 	}
 
+	public static void releaseResourceInstance() {
```

Comment:
Maybe we should keep the same pattern as in the
```java
public static Object getMatchedResource(Message message)
```
method, that is receiving the message by argument, but I am not sure :thinking: . I know this is beautiful, because we encapsulate it inside the method. What about getting rid of the argument in the `getMatchedResource` method?

Reply:
Let's keep the existing pattern. See cb29adc65045dd31703d01f538a12d2bcc67ccf6.